frame/bonded-finance/src/lib.rs

```diff
@@ -306,13 +309,6 @@ pub mod pallet {
 			offer: BondOfferOf<T>,
 			keep_alive: bool,
 		) -> Result<T::BondOfferId, DispatchError> {
-			ensure!(
```

Comment:
so we allow to spam system with near zero offers? i am ok if there will be TODO comment and if others agree for now with that.